src/db/migration/migrate20211116-ZmgDevices.pl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
```

Comment:
I think we should name the script file as migrate20211116-MobileNotificationDevices.pl. so that its clear whats this script is about or for which feature